### PR TITLE
fix: Prompt proper error message for copybook download.

### PR DESCRIPTION
**Pull request by ap891843**
Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
             "properties": {
                 "cobol-lsp.cpy-manager.profiles": {
                     "type": "string",
-                    "description": "Current default profile for copybook downloader"
+                    "description": "Current default profile for copybook downloader\nNOTE:Zowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe"
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/package.json:112]: @zeibura Please check the NOTE section.
image

File: clients/cobol-lsp-vscode-extension/src/constants.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export const DOWNLOAD_QUEUE_LOCKED_ERROR_MSG = `Invalid credentials for profile:
 export const UNLOCK_DOWNLOAD_QUEUE_MSG = "Unblock and retry";
 export const INVALID_CREDENTIALS_ERROR_MSG = `Incorrect credentials in Zowe profile ${PROFILE_NAME_PLACEHOLDER}.`;
 export const PROCESS_DOWNLOAD_ERROR_MSG = "Some copybooks could not be located. Ensure your configuration contains correct paths to copybooks, including nested copybooks. Missing copybooks: ";
-export const SPECIFY_DSN_OR_USS = "Please specify DSNs or USS paths for copybooks in settings.";
+export const PROVIDE_PROFILE_MSG = "Please specify a valid Zowe Explorer profile to download copybooks from the mainframe.";
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/src/constants.ts:29]: @Zeb Please check the error message, when the passed profile is not a valid ZE profile

**Review by temanbrcom — APPROVED** [review]
🌋

**Review by grianbrcom — APPROVED** [review]
:balloon:

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/package.json:112]: I would remove the "NOTE:"

File: clients/cobol-lsp-vscode-extension/src/constants.ts (hunk 2 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/src/constants.ts:29]: that's fine

**Review by zeibura — CHANGES_REQUESTED** [review]
Just remove the "NOTE:"

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json]: updated:
image

**Comment by ap891843** [comment]
> Just remove the "NOTE:"

Updated

**Review by zeibura — APPROVED** [review]

**Pull request merged**